tpu/Dockerfile

```diff
@@ -62,6 +62,9 @@ RUN pip install tensorflow_hub https://storage.googleapis.com/cloud-tpu-tpuvm-ar
 # Tensorflow libtpu:
 RUN curl --output /lib/libtpu.so https://storage.googleapis.com/cloud-tpu-tpuvm-artifacts/libtpu/${TF_LIBTPU_VERSION}/libtpu.so
 
+# Install kagglehub library
+RUN pip install kagglehub
```

Comment:
I noticed we don't clear the cache at the end of each layer which means the image will be larger. Should we add that? Any reason for now doing it like we do for CPU/GPU?

Reply:
We should probably add that, but this image is tiny by comparison already.

Can we move this into the pip install above by just adding it to the list of packages? It can go on its own line at the end.

This image is intended to only have 1 pip install to prevent *any* incompatibilities across the small package set.

Reply:
Makes sense. Done.